### PR TITLE
spectral-cube: init at 0.4.2

### PR DESCRIPTION
**Pull request by smaret**
###### Motivation for this change

This PR provides a package for`spectral-cube`, a Python module for reading and analyzing astrophysical spectral data cubes. It also provides a package for `radio_beam`, on which `spectral-cube` depends.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/radio_beam/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, astropy }:
+
+buildPythonPackage rec {
+  pname = "radio_beam";
+  version = "0.2";
+
+  doCheck = false;
```
Comment (FRidh) [inline pkgs/development/python-modules/radio_beam/default.nix]: Why are the tests disabled? Include a comment in the expression explaining why the tests are disabled.

File: pkgs/development/python-modules/spectral-cube/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+, fetchPypi
+, buildPythonPackage
+, astropy
+, radio_beam }:
+
+buildPythonPackage rec {
+  pname = "spectral-cube";
+  version = "0.4.3";
+
+  doCheck = false;
```
Comment (FRidh) [inline pkgs/development/python-modules/spectral-cube/default.nix]: Why are the tests disabled? Include a comment in the expression explaining why the tests are disabled.

**Review by FRidh — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/radio_beam/default.nix (hunk 1 above)
Comment (smaret) [reply pkgs/development/python-modules/radio_beam/default.nix]: The tests require several `pytest` plugins that are not in `nixpkgs`. I've added a comment in the expression.

Comment (smaret) [reply pkgs/development/python-modules/radio_beam/default.nix]: The tests require several `pytest` plugins that are not in `nixpkgs`. I've added a comment in the expressions.

**Comment by FRidh** [comment]
@GrahamcOfBorg build python2.pkgs.spectral-cube python3.pkgs.spectral-cube python2.pkgs.radio_cube python3.pkgs.radio_cube

**Comment by FRidh** [comment]
Thanks.

**Pull request merged**